### ISSUE TITLE
Import Error for running demo code

### ISSUE THREAD

**Issue by mohsinjuni** [Svito-zar/gesticulator#1]
I have installed required packages with the following command.

`pip install -r requirements.txt`

but got this error:

```
ERROR: torchnlp 0.0.0.1 has requirement scikit-learn==0.20.2, but you'll have scikit-learn 0.22.2.post1 which is incompatible.
ERROR: torchnlp 0.0.0.1 has requirement torch==1.0.0, but you'll have torch 1.4.0 which is incompatible.
ERROR: torchnlp 0.0.0.1 has requirement tqdm==4.28.1, but you'll have tqdm 4.44.1 which is incompatible.

```

Now, if I run demo code with following command:

`python demo.py --audio input/jeremy_howard.wav --text input/jeremy_howard.json
`

I get this error:

```
Traceback (most recent call last):
  File "demo.py", line 5, in <module>
    import torch
  File "/home/<name>/python3.7/site-packages/torch/__init__.py", line 81, in <module>
    from torch._C import *
ImportError: numpy.core.multiarray failed to import

```

Could you please guide on this?

**Comment by Svito-zar** [comment]
Hi @mohsinjuni .

I could not reproduce your issue neither with Python 3.6 nor with Pythor 3.7. 
Never the less I updated the `requirements.txt` to avoid potential issues in the commit https://github.com/Svito-zar/gesticulator/commit/111333e2c201900f530f3717c6bc558521ab7d29.

Please try to install dependencies again and tell me if it works now.
I recommend you to use [conda](https://anaconda.org) virtual environment for it.

**Comment by mohsinjuni** [comment]
Hi @Svito-zar 

Thanks for quickly getting back to me. I am able to reproduce the demo video now. I am listing following things that helped me:

1- torchnlp package error during installation. Since you have already removed this from requirements.txt file, this gets automatically fixed now.
2- I think, I had multiple versions of 'numpy' installed on my machine - one which I already had and one from the requirements.txt file. Because of this, 'torch' was getting confused. So, I removed all the packages which had duplicated installations at '~/miniconda3/lib/python3.7/site-packages/'. Second, I ensured packages are not getting installed from already-downloaded-cached folder by issuing following command:

`pip install --no-cache-dir -r requirements.txt` 

Hope it helps someone else, just in case.
